### PR TITLE
add hls.js integration page

### LINKED ISSUE #91: Add hls.js implementation page
Like the one we had here https://docs.google.com/document/d/12F0Fwl1KYm2ubEEO9PEKzrK0oiAaGGuG2F681PiFoZQ/edit#

### PR DESCRIPTION
**Pull request by mrbar42**
fixes #91

### CODE REVIEW

File: docs/players/hls.js.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Hls.js with Peer5 for HLS integration
+
+<br>
+![](./images/hlsjs.png)
+<br><br>
+[Hls.js](https://github.com/video-dev/hls.js) is an Open source HLS player.
```
Comment (shaharmor) [inline docs/players/hls.js.md]: lowercase O

File: docs/players/hls.js.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+The integration with Peer5 plugin is as easy as it can get.  
+In addition to the player script, include Peer5 client and Hls.js plugin.
```
Comment (shaharmor) [inline docs/players/hls.js.md]: `include Peer5 client and Hls.js plugin.` -> `include the Peer5 client and the Peer5 hls.js plugin.`

File: docs/players/hls.js.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ 
+## Peer5 client and plugins scripts
+add these two scripts to the `head` of your player's page
```
Comment (shaharmor) [inline docs/players/hls.js.md]: Capital A

File: docs/players/hls.js.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Hls.js with Peer5 for HLS integration
+
+<br>
+![](./images/hlsjs.png)
```
Comment (whadar) [inline docs/players/hls.js.md]: @mrbar42 can you upload it?

**Pull request merged**